src/identity/components/OrganizationListTab/OrgBannerPanel.tsx

```diff
@@ -0,0 +1,60 @@
+// Libraries
+import React, {FC} from 'react'
+import {useSelector} from 'react-redux'
+import {BannerPanel, FlexBox, Gradients, IconFont} from '@influxdata/clockface'
+
+// Styles
+import './OrgBannerPanel.scss'
+
+// Constants
+import {CLOUD_URL} from 'src/shared/constants'
+
+// Selectors
+import {selectOrgId} from 'src/identity/selectors'
+
+// Types
+import {OrgAllowanceResponse} from 'src/identity/apis/org'
+
+interface Props {
```

Comment:
Annoyingly the properties have different names, but you've inspired a helpful clarification here (!) 

This type should be at the component level, not in the API layer, since the property names being used here are specific to this component (which unlike other parts of the UI, is relying on the _opposite_ of the value of the `allowed` boolean instead of the `allowed` boolean itself).